### PR TITLE
Env readme

### PR DESCRIPTION
**Pull request by doktoric**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +199,19 @@ After having imported cloudbreak repo root you can launch Datalake application b
 -Ddatalake.cloudbreak.url=http://localhost:8080
 ````
 
+### Running environment in IDEA
+
+After having imported cloudbreak repo root you can launch Environment application by executing the com.sequenceiq.environment.EnvironmentApplication class with the following VM options:
+
+````
+-Denvironment.cloudbreak.url=http://localhost:8080
```
Comment (attilapalfi92) [inline README.md]: a lot of these are already in the application.yml so these vm options are not needed.

https://github.com/hortonworks/cloudbreak/blob/668be14519ad3cd5c152413c7fb26b0b501ac269/environment/src/main/resources/application.yml#L30-L47

File: README.md [hunk 2]
```diff
@@ -195,6 +199,19 @@ After having imported cloudbreak repo root you can launch Datalake application b
 -Ddatalake.cloudbreak.url=http://localhost:8080
 ````
 
+### Running environment in IDEA
```
Comment (attilapalfi92) [inline README.md]: `Running Environment service in IDEA` would be nicer

File: README.md [hunk 3]
```diff
@@ -252,22 +270,36 @@ To run datalake from command line you have to run the below gradle command with
...
 -Dvault.root.token=<TOKEN_FROM_PROFILE_FILE> \
 -Dspring.config.location=$(pwd)/freeipa/src/main/resources/application.yml,$(pwd)/freeipa/build/resources/main/application.properties"
 ````
 
+### Environment
+To run environment service from command line you have to run the below gradle command with the following list of JVM parameters:
+
+````
+./gradlew :environment:bootRun -PjvmArgs="\
+-Denvironment.db.addr=localhost \
```
Comment (attilapalfi92) [inline README.md]: these should go to the application.yml except for the root token

File: README.md [hunk 4]
```diff
@@ -252,22 +270,36 @@ To run datalake from command line you have to run the below gradle command with
 To run freeipa management service from command line you have to run the below gradle command with the following list of JVM parameters:
 
 ````
-./gradlew :freeipa:bootRun -PjvmArgs="-Dfreeipa.db.addr=localhost \
--Dserver.port=8090 \
+./gradlew :freeipa:bootRun -PjvmArgs="\
+-Denvironment.db.port.5432.tcp.addr=localhost \
+-Dserver.port=8087 \
```
Comment (adoroszlai) [inline README.md]: Is FreeIPA service port change intentional?

File: README.md [hunk 5]
```diff
@@ -252,22 +270,36 @@ To run datalake from command line you have to run the below gradle command with
...
 -Dspring.config.location=$(pwd)/freeipa/src/main/resources/application.yml,$(pwd)/freeipa/build/resources/main/application.properties"
 ````
 
+### Environment
+To run environment service from command line you have to run the below gradle command with the following list of JVM parameters:
+
+````
+./gradlew :environment:bootRun -PjvmArgs="\
+-Denvironment.db.addr=localhost \
+-Dserver.port=8090 \
```
Comment (adoroszlai) [inline README.md]: Shouldn't it match the port in [CBD](https://github.com/hortonworks/cloudbreak-deployer/blob/f030ba942d5a496c7bbafe6e801eac38924cd0f7/include/deployer.bash#L407) (ie. 8088)?

**Comment by doktoric** [comment]
closing this one

**Pull request closed without merging**